### DISCUSSION TITLE
WikidPad import?

### DISCUSSION THREAD

**Discussion by wwwald** [TriliumNext/Trilium#1398] [Q&A]
Hi,

I've been using WikidPad for the last 10+ years, but recently gave up on it because of some recurring bugs or problems specific to my setup, that I got fed up with. 

@zadam , I noticed that you used to be an avid WikidPad user before developing Trilium. Some questions:

- do you have scripts/sample code lying around for importing WikidPad wiki's into Trilium?
- would you recommend importing such a huge backlog of notes into Trilium? Should I expect performance issues if I go this route?

**Comment by zadam** [comment]
I remember trying wikidpad out, but I haven't really used it ... So I don't actually have any import scripts.

I want Trilium to be able to handle large amounts of notes (target is 100 000), personally I use it without issue with ~15 000.

**Comment by wwwald** [comment]
Don't remember where I picked up the Wikidpad rumour - probably my imagination then :-)

I may have a go at writing a basic WikidPad importer for my notes. I'd probably go with an external script, writing to the Trilium DB directly. Or do you have suggestions for a better way?

**Reply by zadam** [reply]
I did a quick search and it looks like WikidPad supports HTML export: http://wikidpad.sourceforge.net/help/ItemPublishing.html

You can then ZIP the exported HTML files and import into Trilium ...

**Reply by wwwald** [reply]
That's an awesome start - at least I have all the content now already available. Thanks a lot!
